cmd/podman/libpodruntime/runtime.go

```diff
@@ -48,7 +48,7 @@ func GetRuntime(c *cliconfig.PodmanCommand) (*libpod.Runtime, error) {
 	if c.Flags().Changed("storage-driver") {
 		storageOpts.GraphDriverName = c.GlobalFlags.StorageDriver
 	}
-	if c.Flags().Changed("storage-opt") {
+	if len(c.GlobalFlags.StorageOpts) > 0 {
```

Comment:
Interesting.  I thiought the "Changed" would have handled this.  @baude should we elook at the Changed commands use elsewhere?

Reply:
Changed means that the user specified it, correct?

Reply:
That was my understanding for Changed.